hedera-node/hedera-app/src/main/java/com/hedera/node/app/authorization/PrivilegesVerifier.java

```diff
@@ -234,7 +197,7 @@ private SystemPrivilege checkCryptoUpdate(
         final var targetId = op.accountIDToUpdateOrElse(AccountID.DEFAULT);
         final long targetNum = targetId.accountNumOrElse(0L);
         final var treasury = accountsConfig.treasury();
-        final var payerNum = payerId.accountNumOrThrow();
+        final var payerNum = payerId.accountNumOrElse(0L);
```

Comment:
We should not do this IMO, unless we have a check somewhere that `treasury` and `accountsConfig.systemAdmin()` can never be 0. We should return `UNAUTHORIZED` if the `accountNum` is not set.